src/data/real/nnreal.lean

```diff
@@ -130,6 +130,9 @@ by rw [← nnreal.coe_one, nnreal.coe_eq]
 
 lemma coe_ne_zero {r : ℝ≥0} : (r : ℝ) ≠ 0 ↔ r ≠ 0 := by norm_cast
 
+lemma eq_zero_or_pos (r : ℝ≥0) : r = 0 ∨ 0 < r :=
+by { rw eq_comm, exact eq_or_lt_of_le r.2 }
```

Comment:
This is true more generally as
```
lemma eq_zero_or_pos {R} (r : R) [canonically_ordered_add_monoid R] : r = 0 ∨ 0 < r :=
(zero_le r).eq_or_lt.imp_left eq.symm
```

Reply:
Oh nice, I'll think about where to move it.

Reply:
I put it...in the root namespace!